merkletree/proof.go

```diff
@@ -88,13 +89,22 @@ func VerifyAuthPath(ap *AuthenticationPath,
 	return true
 }
 
+func VerifyCommitment(salt []byte, key string, value []byte, commitment []byte) bool {
+	got := crypto.Digest(salt, []byte(key), value)
```

Comment:
I remember early on I told you to remove the `crypto.Commitment` function because it was redundant with `Digest`.  I sort of regret that now :(